INSTALL.md

```diff
@@ -24,56 +20,142 @@
 ## Installation
 
 ### Deploy the Container
-1. Connect to the RPi, via SSH for example.
-2. 
-    a. Clone the repo.
-
- **OR**
-
-2.
-    b. Create a `tesla_ble_mqtt_docker` folder in your user directory: `cd ~ && mkdir tesla_ble_mqtt_docker && cd $_.`
-
-    Add a docker-compose.yml file: `nano docker-compose.yml`.
-4. Adjust `docker-compose.yml` to your needs. Pay attention to the environment variables :
+There are various methods to deploy the container, I describe the main ones below:
+#### Deploy using Command Line ####
+1. Create a tesla_ble_mqtt_docker folder in your user directory and change directory into it:
    ```yaml

```

Comment:
yaml or shell?

Reply:
I used what the OP did on the original version. It is shell, I will update